src/app/(account)/settings/Settings.tsx

```diff
@@ -1,21 +1,24 @@
 'use client';
 
-import type { MembershipPayment } from '@/server/verify-membership-payment';
 import { useRouter } from 'next/navigation';
 import React, { useState } from 'react';
 import Sidebar from './Sidebar';
+import type { SettingData } from './page';
+import AccountSettings from './tabs/AccountSettings';
 import MembershipSettings from './tabs/MembershipSettings';
+import PersonalInfoSettings from './tabs/PersonalInfoSettings';
 
 export const TAB_NAMES = ['Account', 'Personal Info', 'Membership', 'Notifications'] as const;
```

Comment:
Maybe make the "Membership" tab go first in the sidebar since that's the default tab.